polaris-react/src/components/EmptyState/tests/EmptyState.test.tsx

```diff
@@ -8,6 +8,7 @@ import {Text} from '../../Text';
 import {UnstyledLink} from '../../UnstyledLink';
```

Comment:
It's not showing up on the diff but the `Stack` import on line 6 of this file needs to be removed to resolve the build errors in ci check